internal/pkg/vm/internal/runtime/runtime.go

```diff
@@ -68,8 +68,12 @@ type InvocationContext interface {
 // and ensure the context implementation exposes them.
 type ExtendedInvocationContext interface {
 	InvocationContext
-	// Dragons: add new CreateActor (this is just for the init actor)
-	// CreateActor(code cid.Cid, params []interface{}) address.Address
+	// Create an actor in the state tree.
+	//
+	// This will determine an address for the actor and call its `Constructor()` method.
+	//
+	// WARNING: May only be called by InitActor.
```

Comment:
Document which address is returned, the id-address, or the reorg-stable address. Maybe we should return both.

Reply:
Good point.